### PR TITLE
chore: config value updates and bug fixes

### PR DESCRIPTION
**Pull request by dhfang**

### CODE REVIEW

File: hyperlane/cosmos/validator_announce.go [hunk 1]
```diff
@@ -78,6 +78,9 @@ func (va *ValidatorAnnounceQuerier) GetAnnouncedValidatorStorageLocations(ctx co
 		if !ok {
 			return nil, fmt.Errorf("got unexpected type for second element of validator storge location, exepcted []any")
 		}
+		if len(locationsAny) == 0 {
```
Comment (dhfang) [inline hyperlane/cosmos/validator_announce.go:81]: can panic if no storage location

File: ordersettler/ordersettler.go [hunk 2]
```diff
@@ -137,26 +137,11 @@ func (r *OrderSettler) findNewSettlements(ctx context.Context) error {
 			}
 
 			// ensure order exists on source chain
-			exists, err := sourceBridgeClient.OrderExists(ctx, sourceGatewayAddress, fill.OrderID, big.NewInt(int64(height)))
+			exists, amount, err := sourceBridgeClient.OrderExists(ctx, sourceGatewayAddress, fill.OrderID, big.NewInt(int64(height)))
 			if err != nil {
 				return fmt.Errorf("checking if order %s exists on chainID %s: %w", fill.OrderID, sourceChainID, err)
 			}
 			if !exists {
-				// if the order does not exist on the source, set the
```
Comment (dhfang) [inline ordersettler/ordersettler.go:145]: The orders table should be considered logically separate from the order_fills table. There may not be an orders entry for every order_fills entry.

File: ordersettler/ordersettler.go [hunk 3]
```diff
@@ -166,6 +151,7 @@ func (r *OrderSettler) findNewSettlements(ctx context.Context) error {
 				SourceChainGatewayContractAddress: sourceGatewayAddress,
 				OrderID:                           fill.OrderID,
 				SettlementStatus:                  dbtypes.SettlementStatusPending,
+				Amount:                            amount.String(),
```
Comment (dhfang) [inline ordersettler/ordersettler.go:154]: Cant calculate the batch settlement value without amount

File: ordersettler/ordersettler.go [hunk 4]
```diff
@@ -218,12 +204,16 @@ func (r *OrderSettler) settleOrders(ctx context.Context) error {
 // verifyOrderSettlements checks on all instated settlements and updates their
 // status in the db with their on chain tx results.
 func (r *OrderSettler) verifyOrderSettlements(ctx context.Context) error {
+	pendingSettlements, err := r.db.GetAllOrderSettlementsWithSettlementStatus(ctx, dbtypes.SettlementStatusPending)
```
Comment (dhfang) [inline ordersettler/ordersettler.go:207]: verifyOrderSettlement is responsible for updating settlement state from pending to initiated. probably worth refactoring the functionality here for more clarity in the future.

**Comment by nadim-az** [comment]
one small nit, otherwise lgtm.

ty for the bug fixes 🙏

**Review by nadim-az — APPROVED** [review]

**Pull request merged**